### ISSUE TITLE
Cannot display pipelinerun 

### ISSUE THREAD

**Issue by afrittoli** [tektoncd/dashboard#1200]
# Expected behavior
I should be able to display any pipelinerun

# Actual behavior
I found at least [one pipelinerun](http://dashboard.robotcat.tekton.dev/#/namespaces/tutorial/pipelineruns/tutorial-pipeline-run-1) that does not work:

<img width="383" alt="image" src="https://user-images.githubusercontent.com/2205608/77757390-14040800-7029-11ea-9f72-27453375919e.png">

This is the pipeline from the pipeline project tutorial.

# Steps to reproduce the problem

1. Pipeline definitinon: https://github.com/tektoncd/pipeline/blob/master/docs/tutorial.md#creating-and-running-a-pipeline
1. View the pipeline through the dashboard http://dashboard.robotcat.tekton.dev/#/namespaces/tutorial/pipelines/tutorial-pipeline
1. View the pipeline run through the dashboard 
1. http://dashboard.robotcat.tekton.dev/#/namespaces/tutorial/pipelineruns/tutorial-pipeline-run-1

# Environment
- Kubernetes Platform:

```
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.2", GitCommit:"c97fe5036ef3df2967d086711e6c0c405941e14b", GitTreeState:"clean", BuildDate:"2019-10-15T23:41:55Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"15+", GitVersion:"v1.15.9-gke.26", GitCommit:"525ce678faa2b28483fa9569757a61f92b7b0988", GitTreeState:"clean", BuildDate:"2020-03-06T18:47:39Z", GoVersion:"go1.12.12b4", Compiler:"gc", Platform:"linux/amd64"}
```

- Tekton Pipelines version:
Nightly: v20200327-0df48d48e5

- Tekton Triggers version:
Nightly: v20200327-a0fd950080

- Tekton Dashboard version:
Beta release https://console.cloud.google.com/storage/browser/tekton-releases/dashboard/adambeta4/adambeta4/previous/beta/?project=tekton-releases

# Additional Info

**Comment by afrittoli** [comment]
Uhm, probably a duplicate of https://github.com/tektoncd/dashboard/issues/1198 - however I'm using the dashboard beta release here, so I think this needs to be fixed before we can release?

**Comment by AlanGreene** [comment]
@afrittoli yes it's a duplicate. There are still a few changes that need to go in before we release 0.6 for the beta.

Closing this and we can keep #1198 for tracking.
